packages/contracts/contracts/MarketRegistry.sol

```diff
@@ -37,7 +37,7 @@ contract MarketRegistry is
         bool lenderAttestationRequired;
         EnumerableSet.AddressSet verifiedLendersForMarket;
         mapping(address => bytes32) lenderAttestationIds;
-        uint32 paymentCycleDuration; //unix time
+        uint32 paymentCycleValue; // unix time (seconds) OR day of the month (1-31) - depending on the `paymentCycleType`
```

Comment:
lets actually revert this back and if the `paymentCycleType == Monthly` then this should be forced to be set to `30 days`

we can revert the name of `Value` back to `Duration` in all cases